app/validators/return-requirements/abstraction-period.validator.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+/**
+ * Validates data submitted for the `/return-requirements/{sessionId}/abstraction-period` page
+ * @module AbstractionPeriodPresenter
+ */
+
+const Joi = require('joi').extend(require('@joi/date'))
+
+const { leftPadZeroes } = require('../../presenters/base.presenter.js')
+
+/**
+ * Validates data submitted for the `/return-requirements/{sessionId}/abstraction-period` page
+ *
+ * When setting up a requirement, users must specify an abstraction period for the return requirement. Users must input a valid from and to
```

Comment:
'When setting up a return requirement, users need to specify an abstraction period. For this, the users must input a valid `from` and `to` date. If a user does not enter valid dates the validation will return an error.'

I would reword this comment. The comment on line 15 goes over the line length and in parts of the comment it doesn't read properly.